### ISSUE TITLE
generator-star-spacing - Unexpected space after *

### ISSUE THREAD

**Issue by FredyC** [babel/eslint-plugin-babel#49]
I have a following code with default rule setting ("before").

``` js
async lockCallback(normalizedProfile, token) {
}
```

Fails with `Unexpected space after *  babel/generator-star-spacing`

Using "after" fixes first issue, but causes another two for the following code

``` js
export function *iterateChildren() {
}
```

```
warning  Unexpected space before *  babel/generator-star-spacing
warning  Missing space after *      babel/generator-star-spacing
```

Using "both" or "neither" is unwanted, but essentially fixes the first issue.

>    "eslint": "~1.9.0",
>    "eslint-plugin-babel": "^3.1.0",
>    "eslint-plugin-react": "^3.16.1",
>    "babel": "~5.8.23",
>    "babel-core": "~5.8.34",
>    "babel-eslint": "~4.1.4",

**Comment by hzoo** [comment]
deprecating those rules so closing https://github.com/babel/eslint-plugin-babel/pull/115
